cypress/e2e/context-selection/period/index.js

```diff
@@ -1,4 +1,10 @@
-import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+import { Given, When, Then } from '@badeball/cypress-cucumber-preprocessor'
+
+// It seems that the new cypress cucumber library splits different steps into
+// different test cases. Cypress does not persist aliases across tests,
+// mutable seem to be working though
+let selectedPeriodLabel = ''
+let selectedPeriodValue = ''
```

Comment:
This should still be possible from what I could find:

- https://github.com/badeball/cypress-cucumber-preprocessor/discussions/959
- https://github.com/badeball/cypress-cucumber-preprocessor/pull/74

I think a mutable value used in this way is an easy way to have tests accidentally affect eachother. Using as should be possible so that seems preferable to me.

Reply:
Unless something has changed in the meantime, but I couldn't find any docs confirming that.

Reply:
These issues all seem to confirm to me that it should work:

- https://github.com/badeball/cypress-cucumber-preprocessor/issues/246
- https://github.com/badeball/cypress-cucumber-preprocessor/issues/551
- https://github.com/badeball/cypress-cucumber-preprocessor/issues/471
- https://github.com/badeball/cypress-cucumber-preprocessor/issues/413
- https://github.com/badeball/cypress-cucumber-preprocessor/issues/374
- https://github.com/badeball/cypress-cucumber-preprocessor/issues/246

Hopefully there's something in there that helps resolve the aliasing issue